### PR TITLE
added lastIndexOf() javascript question

### PR DESCRIPTION
**Pull request by KristenHarman**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Adds to #14 

Question is about lastIndexOf() method.

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1783,7 +1783,18 @@ const javascriptQuiz = [
...
+  },
+  {
+    Question:
+      "In JavaScript, the ___________ method returns the index position of the last occurrence of a value in an array.",
+    Answer: "lastIndexOf()",
+    Distractor1: "indexOf()",
+    Distractor2: "charAt(0)",
+    Distractor3: "getPosition()",
+    Explanation:
+      "The lastIndexOf() method returns the last index at which a given element can be found in the array, or -1 if it is not present. The array is searched backwards, starting at fromIndex.",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:1795]: I think it is ok to drop the last sentence. If users want to learn more, then they click on the learn more link in the quiz 👍 

```suggestion
      "The lastIndexOf() method returns the last index at which a given element can be found in the array, or -1 if it is not present.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/javascript-quiz.ts (hunk 1 above)
Comment (KristenHarman) [reply src/data/javascript-quiz.ts:1795]: I was thinking that, as well. Thank you for letting me know.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**